contracts/contracts/MACI.sol

```diff
@@ -98,7 +98,7 @@ contract MACI is IMACI, DomainObjs, Params, Utilities, Ownable {
     InitialVoiceCreditProxy _initialVoiceCreditProxy,
     TopupCredit _topupCredit,
     uint8 _stateTreeDepth
-  ) {
+  ) payable {
```

Comment:
Sorry, not quite sure why we need payable for the constructor?

Reply:
Allows to save a tiny bit of gas while deploying a contract: https://github.com/OpenZeppelin/openzeppelin-contracts/issues/3059

Reply:
This is a cool feature to me, since I searched and also asked ChapGPT if we could make constructor payable and find nothing 🤔 Thanks for the information~~

Reply:
no worries 😄